src/new/wallet.js

```diff
@@ -325,15 +325,52 @@ class HathorWallet extends EventEmitter {
       }],
     };
 
+    return this.completeTxData(data, txToken);
+  }
+
+  /**
+   * Complete transaction data with inputs
+   *
+   * @param {Object} data Partial data that will be completed with inputs
+   * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
+   *
+   * @return {Object} Object with 'success' and completed 'data' in case of success, and 'message' in case of error
+   **/
+  completeTxData(partialData, token) {
+    const txToken = token || this.token;
     const walletData = wallet.getWalletData();
     const historyTxs = 'historyTransactions' in walletData ? walletData.historyTransactions : {};
-    const ret = wallet.prepareSendTokensData(data, txToken, true, historyTxs, [txToken]);
+    const ret = wallet.prepareSendTokensData(partialData, txToken, true, historyTxs, [txToken]);
+
+    return ret
+  }
 
-    if (!ret.success) {
-      return ret;
+  /**
+   * Send a transaction from its outputs
+   *
+   * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value'}
+   *
+   * @return {Promise} Promise that resolves when transaction is sent
+   **/
+  sendManyOutputsTransaction(outputs) {
```

Comment:
Isn't this the same as `sendMultiTokenTransaction`?

Reply:
No. The `sendMultiTokenTransaction` is more than this one and has lot of workarounds that I don't like (e.g. manually update outputs with fixed values). It was done in a hurry for the twitter bot and it's not well tested. I was about to delete this method but I decided to leave it there and keep just the new method in the PR.